pkg/action/action.go

```diff
@@ -200,7 +200,11 @@ func (cfg *Configuration) renderResources(ch *chart.Chart, values chartutil.Valu
 
 	for _, m := range manifests {
 		if outputDir == "" {
-			fmt.Fprintf(b, "---\n# Source: %s\n%s\n", m.Name, m.Content)
+			if hideSecret && m.Head.Kind == "Secret" && m.Head.Version == "v1" {
```

Comment:
Should we be specific about the group, here? Would it be a surprise to have a Secret CRD and have its output suppressed?

Reply:
The `Version` property holds the group and version. For example, for deployments it's `apps/v1`. The `Head` is a `SimpleHead` type that has some information in a simplified form. This should only catch the core Secret.

The output is completely suppressed because it's validated for yaml and openapi schema. It's hard to change the output to something like `[HIDDEN]` which would make it fail both of those. Someone had to opt-in for it to be hidden.